PowerPointLabs/PowerPointLabs/NativeMethods.cs

```diff
@@ -344,6 +366,17 @@ internal enum MagnifierFilterMode : int
             MW_FILTERMODE_INCLUDE = 1
         }
 
+        internal enum WindowLong : int
+        {
+            GWL_WNDPROC = -4,
+            GWL_HINSTANCE = -6,
+            GWL_HWNDPARENT = -8,
+            GWL_ID = -12,
+            GWL_STYLE = -16,
+            GWL_EXSTYLE = -20,
+            GWL_USERDATA = -21

```

Comment:
I would suggest keeping it in uppercase (same as other enum in Native class) to keep it the same as the ones defined in Winuser.h
https://msdn.microsoft.com/en-us/library/windows/desktop/ms633591(v=vs.85).aspx